thumbor/point.py

```diff
@@ -30,7 +34,7 @@ def to_dict(self):
         }
 
     @classmethod
-    def from_dict(cls, values):
+    def from_dict(cls: Type[T], values: dict) -> T:
```

Comment:
Here you can use `-> FocalPoint`.

Reply:
When I use FocalPoint I get this error message:
```
/thumbor/point.py:62:10: F821 undefined name 'FocalPoint'
```

Should I ignore it too ?